src/main/java/seedu/guest/ui/GuestCard.java

```diff
@@ -54,14 +59,112 @@ public GuestCard(Guest guest, int displayedIndex) {
         this.guest = guest;
         id.setText(displayedIndex + ". ");
         name.setText(guest.getName().fullName);
+        name.setWrapText(true);
```

Comment:
Just a suggestion, I see a couple of repeated fields, would it be possible to have there was a method created to build these ui components?

Reply:
Agreed. Abstracted into a method.